ecl/resource_ecl_mlb_policy_v1.go

```diff
@@ -381,12 +500,36 @@ func resourceMLBPolicyV1UpdateConfigurations(d *schema.ResourceData, client *ecl
 			updateStagedOpts.DefaultTargetGroupID = &defaultTargetGroupID
 		}
 
+		if d.HasChange("backup_target_group_id") {
+			isConfigurationsUpdated = true
+			backupTargetGroupID := d.Get("backup_target_group_id").(string)
+			updateStagedOpts.BackupTargetGroupID = &backupTargetGroupID
+		}
+
 		if d.HasChange("tls_policy_id") {
 			isConfigurationsUpdated = true
 			tlsPolicyID := d.Get("tls_policy_id").(string)
 			updateStagedOpts.TLSPolicyID = &tlsPolicyID
 		}
 
+		if d.HasChange("server_name_indications") {
+			isConfigurationsUpdated = true
+			serverNameIndications := make([]policies.UpdateStagedOptsServerNameIndication, len(d.Get("server_name_indications").([]interface{})))
```

Comment:
[NITS]
細かくて申し訳ないのですが、空行を入れて頂けると助かります

```suggestion
			isConfigurationsUpdated = true

			serverNameIndications := make([]policies.UpdateStagedOptsServerNameIndication, len(d.Get("server_name_indications").([]interface{})))
```

Reply:
空行を入れるよう修正しました